config/default.yml

```diff
@@ -76,7 +76,23 @@ RSpec/ContextWording:
 RSpec/DescribeClass:
   Description: Check that the first argument to the top-level describe is a constant.
   Enabled: true
+  IgnoredMetadata:
+    type:
```

Comment:
By the way, it seems to be possible to put a symbol if you surround it by colons:
```yaml
IgnoredMetadata:
  :type:
    - channel
```
Should be sufficient for keys.

Reply:
By doing that I get the following error:
```
Failure/Error: CONFIG         = YAML.safe_load(CONFIG_DEFAULT.read).freeze

Psych::DisallowedClass:
  Tried to load unspecified class: Symbol
```
 The method can be configured to allow symbols, but I couldn't find anywhere why it's unsafe by default in the first place. What do you think?

Reply:
Oh, bummer, didn't know symbols are now allowed. 👍